### PR TITLE
Website clean up post migration

### PR DESCRIPTION
**Pull request by shifucun**
* Remove reference to datcom-store-dev
* Remove old graph browser code
* Remove unused API
* Minor style update

### CODE REVIEW

**Review by chejennifer — APPROVED** [review]
thanks for the clean up!

File: server/routes/api/browser.py [hunk 1]
```diff
@@ -70,15 +63,15 @@ def get_property_labels(dcid):
...
-    date_selector = "?obsDate"
+    date_selector = " ?obsDate"
     if date:
         date_triple = f'?svObservation observationDate "{date}" .'
         date_selector = ""
     sparql_query = f"""
-        SELECT ?dcid ?mmethod ?obsPeriod {date_selector}
-        WHERE {{ 
+        SELECT ?dcid ?mmethod ?obsPeriod{date_selector}
+        WHERE {{
```
Comment (beets) [inline server/routes/api/browser.py:72]: are these whitespaces significant (i.e. should we leave the ones after "." in the WHERE clause behind?)

File: server/tests/api_browser_test.py [hunk 2]
```diff
@@ -38,10 +38,10 @@ def test_required_predicates(self):
     @patch('routes.api.browser.dc.query')
     def test_observation_node_dcid_returned(self, mock_query):
         expected_query = '''
-        SELECT ?dcid ?mmethod ?obsPeriod 
-        WHERE { 
+        SELECT ?dcid ?mmethod ?obsPeriod
```
Comment (beets) [inline server/tests/api_browser_test.py:41]: ditto comment here re: whitespace significance

**Review by beets — APPROVED** [review]
thx for the cleanup!

File: server/routes/api/browser.py (hunk 1 above)
Comment (shifucun) [reply server/routes/api/browser.py:72]: This is ok as the query is a multi-line string literal. Also verified locally.

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**